https://issues.redhat.com/browse/ACM-25145

dockerymick opened the pull request.

dockerymick committed:
https://issues.redhat.com/browse/ACM-25145

dockerymick committed:
updates after review from dev, confirm via slack

dockerymick committed:
Removed a known issue that was dixed, confirmed by Randy

dockerymick commented on release_notes/known_issues_console.adoc:
```diff
@@ -95,21 +95,14 @@ To work around the issue, zoom out by using your browser zoom controls, increase
 You must log out of the external identity provider before you attempt to log out of {acm-short}.
 
-//this needs an issues attached if we want to keep it, please see the dev team if you cannot find this issue.
+//this needs an issues attached if we want to keep it, please see the dev team if you cannot find this issue. (MJ checking)
```
@JustinKuli can you help confirm if this issue exists? In the history for this file, you and Matt helped with adding it.

JustinKuli replied:
I think we could remove this from "known issues". The original report was https://github.com/stolostron/backlog/issues/13719, which states that the same issue also affects openshift itself. By now I feel like there's a good chance it's just fixed

dockerymick replied:
Thanks Justin! i would assume that as well. I'm not sure who to check with about the issue, bu tsince Derrick created the issue, I'll ask if he knows where that OpenShift issue is. Regardless, I will remove this known issue in 2.12-2.15.

dockerymick committed:
Remove known issue about logout from ACM console

Removed the known issue regarding logging out from the ACM console when using an external identity provider.
